ci/kind/test-e2e-kind.sh

```diff
@@ -132,9 +132,9 @@ function run_test {
   fi
   sleep 1
   if $coverage; then
-      go test -v -timeout=50m github.com/vmware-tanzu/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --coverage --coverage-dir $ANTREA_COV_DIR
+      go test -v -timeout=70m github.com/vmware-tanzu/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR --coverage --coverage-dir $ANTREA_COV_DIR
   else
-      go test -v -timeout=45m github.com/vmware-tanzu/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR
+      go test -v -timeout=65m github.com/vmware-tanzu/antrea/test/e2e -provider=kind --logs-export-dir=$ANTREA_LOG_DIR
```

Comment:
can we run the legacy tests only on jenkins and disable them in Kind?

Reply:
agree

Reply:
ok.

Reply:
A new issue, even the legacy tests are skipped in Kind, 50m is not enough for other tests.